ow_materials/include/MaterialDatabase.h

```diff
@@ -0,0 +1,42 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef MATERIAL_DATABASE_H
+#define MATERIAL_DATABASE_H
+
+#include <string>
+#include <ostream>
+#include <unordered_map>
+
+#include <Material.h>
+
+namespace ow_materials
+{
+
+class MaterialDatabase
+{
+public:
+  MaterialDatabase() = default;
+  ~MaterialDatabase() = default;
+
+  MaterialDatabase(const MaterialDatabase&) = delete;
+  MaterialDatabase& operator=(const MaterialDatabase&) = delete;
+
+  void addMaterial(Material mat);
```

Comment:
Seems that argument would be better as const&.

Reply:
done